docs/release/release-process.md

```diff
@@ -192,6 +201,77 @@ success, all release artifacts will be officially published.
 After the release build succeeds, verify that the correctly versioned Stack images have been pushed
 to Docker Hub.
 
+### Template Stack Release Process
+
+The Template Stacks we maintain are slightly different from the
+controller-based stacks that we maintain. Their processes are similar
+but a little simpler. This section will walk through how to release the
+Template Stacks themselves, and does not directly apply to core
+Crossplane.
+
+For Template Stacks, we do not use release branches unless we need to
+make a patch release. In the future we may need a more robust branching
+strategy, but for now we are not using branches because it is simpler.
+
+Note that Template Stacks **do not** require any code changes to update
```

Comment:
Actually if we release a new version of [`templating-controller`](https://github.com/crossplane/templating-controller), we'd probably want to update the [`controllerImage`](https://github.com/crossplane/stack-minimal-gcp/blob/master/.registry/behavior.yaml#L8) in the template stack to the latest version.

Reply:
We should also be noting that a release of the `templating-controller` should result in an update to `cluster/charts/crossplane-controllers/values.yaml.tmpl` and `docs/install-crossplane.md`.

Reply:
In terms of timing, it may be advisable to release templating-controller ahead of crossplane when coordinating same-day releases.

Reply:
Yeah. This is all true, but it's something that I'm thinking should be documented on the templating-controller side. We've done a pretty good job of decoupling templating-controller releases from crossplane releases already, so I'm thinking we should keep it that way. So I would put this in documentation on the templating-controller side so that we see it when we're releasing the templating-controller.

Reply:
I've added some notes discussing the templating-controller project

Reply:
Thank you for mentioning `templating-controller` but would you mind adding a note to say something like `check the latest version of templating-controller and update to it in behavior.yaml` ? At least, the releaser would make sure that template stacks do use the latest version before the release.

Reply:
Hmm double checking that is a good idea . . . we may want them to check it in the crossplane helm chart too.

Reply:
i'll add a note for this in my #1367 PR @suskin and @muvaf